docs/getting_started.rst

```diff
@@ -0,0 +1,79 @@
+
+.. py:currentmodule:: rticonnextdds_connector
+
+Getting Started
+===============
+
+Installing RTI Connector for Python
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Make sure you have Python installed. Then use *pip* to install *RTI Connector*
```

Comment:
Should we be more specific than Python (the specific compatible versions)

Reply:
Or just say, Make sure you have a compatible version of Python installed

Reply:
We list the python versions we've tested below